src/service/apiClient.js

```diff
@@ -72,8 +72,9 @@ export {
     getUsers,
     register,
     createProfile,
-    get,
```

Comment:
Be careful about adding changes that are unnecessary - this causes everyone to have to merge these changes.

If this is unnecessary, please remove this change from the PR

Reply:
Without line 75 the code gets errors and doesn't render

Reply:
Interesting. Can you reproduce the error and show me?

Reply:
![image](https://github.com/boolean-uk/team-dev-client-c10/assets/124312512/75d79db6-6551-4709-b5ef-0f36e39efff2)